index.html

```diff
@@ -10,9 +10,14 @@
   # Customize your project's badges. Delete any entries that do not apply.
   custom:
     - name: Azure Spring Boot (GitHub)
```

Comment:
Update name to "Spring Cloud Azure (GitHub)" since the pointing GitHub repo has changed ?